app/Contact.php

```diff
@@ -1,15 +1,16 @@
 <?php
 
-namespace App\Models;
+namespace App;
 
```

Comment:
Fix namespace, we're having this error: Class App\Models\Contact located in ./app/Contact.php does not comply with psr-4 autoloading standard. Skipping.